src/main/java/nextstep/subway/applicaion/StationService.java

```diff
@@ -47,4 +48,10 @@ public void deleteStationById(Long id) {
     private StationResponse createStationResponse(Station station) {
         return StationResponse.of(station);
     }
+
+    public Station findById(Long id) {
+        return stationRepository.findById(id).orElseThrow(() -> {
```

Comment:
```suggestion
        return stationRepository.findById(id)
                .orElseThrow(NotFoundException::new);
```
orElseThrow에서 throw를 안 써도 되지 않나요?